### ISSUE TITLE
Avoid triggering the downloads panel when STG backups are run in the background (Firefox 98+)

### ISSUE THREAD

**Issue by rubcap** [Drive4ik/simple-tab-groups#939]
**Is your feature request related to a problem? Please describe.**
When the backup function in STG is enabled, it apparently triggers the download panel in Firefox v98+ to open as well. However, the panel doesn't show the created STG .JSON file, only older downloads. Firefox v98+ users using STG with the backup function enabled may be confused why the panel opens seemingly randomly.

**Describe the solution you'd like**
STG backups don't trigger the download panel in Firefox _at all_. This is a background process that doesn't need my attention.
And if they _really_ must trigger the panel, I preferably want the panel to show the name/folder of the created JSON file.

**Describe alternatives you've considered**
- Disabling the backup function in STG.
- Manually setting `browser.download.alwaysOpenPanel` in `about:config` to FALSE.

**Additional context**
Firefox v98 introduced a change in download management, making the download panel pop up after each finished download (unless you set `browser.download.alwaysOpenPanel` in `about:config` to FALSE) and making more use of the OS's default "Downloads" folder. This idea is to let users find and open their downloads more easily and interruption-free (although one could argue that popping up the panel every time is actually introducing interruptions?).

_Thx in advance. Love your addon!_

**Comment by SpaceClicker** [comment]
Wow thanks!

My download panel kept opening at "random" times and I had no idea why, it was getting really annoying.  Now I know why.  It's strange that the panel itself does not show any downloads, so there is no clue that it is STG making a backup in the background.  I think I also have other add-ons that are saving stuff at regular scheduled intervals, so I guess they are also causing this issue...

**Comment by s0m3guy** [comment]
Yes, this is very annoying. Like @SpaceClicker said I thought this appears to be random but at some point I saw that there was a download finishing with something like "STG" in the name and therefore concluded it had to be the STG backup.

**Comment by ShingenPizza** [comment]
I can confirm it is caused by STG, i was recording my screen while it happened and it was one of the auto backup filenames that popped out for a moment, and then disappeared.

**Comment by nekohayo** [comment]
Yeah, but that's not STG's fault, Mozilla is thinking about how to solve that. See https://bugzilla.mozilla.org/show_bug.cgi?id=1759231

**Comment by bryanpedini** [comment]
Now happily following this (and bug 1759231) to get this annoyance sorted out.

**Comment by rddunphy** [comment]
Looks like this is fixed in FF101, which should be released at the end of May.

**Comment by bryanpedini** [comment]
> Looks like this is fixed in FF101

FF 103.0.2 (64-bit) user here. issue still persists...
